### ISSUE TITLE
Generate a runtime error when there is an error

### ISSUE THREAD

**Issue by fu-sen** [someone-stole-my-name/cfdns#1]
For example, an error will occur if there was no A zone originally,
At that time, no error is displayed and a runtime error occurs.

From the line number displayed at that time,, I just feel like `log.Println` is missing. 

Note that I am using Windows 10.

**Comment by someone-stole-my-name** [comment]
You are right, it shouldn't cause a runtime error. But just to be clear, `"Entry": "myzone.xyz"` should be an A record in `"Zone": "myzone.xyz"`.

**Comment by fu-sen** [comment]
Yes, my cfdns is already working properly. Since this is an issue, I am reporting bugs only.

**Comment by fu-sen** [comment]
This is what my Windows 10 got:
```
PS> ./cfdns
2020/01/19 23:08:42 Using:  config.json
2020/01/19 23:08:43 Updating record "example.net.eu.org" from zone "example.net.eu.org"
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.main()
        C:/Users/Christian/Documents/GitHub/cfdns/cfdns.go:79 +0xc0b
```
Occurs in the following cases:
- Username, API-Key, Zone and Entry is correct
- Specified Zone uses CNAME instead of A

Since I checked and confirmed that it was giving the error correctly,
This may be the case you did not expect.

**Comment by someone-stole-my-name** [comment]
Fixed in d51cce19e000551b1af92e97f28d43f434c2bf86
